package.json

```diff
@@ -41,7 +41,8 @@
     "safe-buffer": "5.2.1",
     "tweetnacl": "1.0.3",
     "valid-url": "1.0.9",
-    "web3-utils": "1.6.0"
+    "web3-utils": "1.6.0",
+    "ethereumjs-util":"7.1.3"
```

Comment:
:information_source: this package was already in our dependency tree through web3-utils (we use its `isAddress`)

Reply:
yes , i find that isAddress different , `ethereumjs-util` just check length and hex , `web3-utils` also check sum , so i use web3 isAddress

[ethereumjs-util checkAddress](https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/util/src/account.ts#L128)
[web3-utils checkAddress](https://github.com/ChainSafe/web3.js/blob/1.x/packages/web3-utils/src/utils.js#L85)